tiledb/sm/serialization/tiledb-rest.capnp

```diff
@@ -398,27 +398,34 @@ struct ReadState {
   # The subarray partitioner
 }
 
-struct ConditionClause {
-  # A clause within a condition
+struct ASTNode {
```

Comment:
These are breaking changes, we can't remove the old cap'n proto structures. We need to leave the `ConditionClause` to support backwards compatibility.